### PR TITLE
diffoscope: add page

### LINKED ISSUE #4644: page request: diffoscope
For the [diffoscope](https://diffoscope.org/) command line utility.

### PR DESCRIPTION
**Pull request by gutjuri**
closes #4644

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/diffoscope.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`diffoscope --no-progress {{path/to/file1}} {{path/to/file2}}`
+
+- Compare two files and write a HTML-report to a file or stdout:
+
+`diffoscope --html {{path/to/outfile|-}} {{path/to/file1}} {{path/to/file2}}`
+
+- Compare two directories excluding files with a name matching a specified pattern:
+
+`diffoscope --exclude {{pattern}} {{path/to/dir1}} {{path/to/dir2}}`
```
Comment (navarroaxel) [inline pages/common/diffoscope.md:20]:
```suggestion
`diffoscope --exclude {{pattern}} {{path/to/directory1}} {{path/to/directory2}}`
```

File: pages/common/diffoscope.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Compare two directories and control whether directory metadata is considered:
+
+`diffoscope --exclude-directory-metadata {{auto|yes|no|recursive}} {{path/to/dir1}} {{path/to/dir2}}`
```
Comment (navarroaxel) [inline pages/common/diffoscope.md:24]:
```suggestion
`diffoscope --exclude-directory-metadata {{auto|yes|no|recursive}} {{path/to/directory1}} {{path/to/directory2}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @gutjuri Just a few comments.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/diffoscope.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# diffoscope
+
+> Comparison of files, archives and directories.
```
Comment (sbrl) [inline pages/common/diffoscope.md:3]: Perhaps this will read better?
```suggestion
> Compare files, archives, and directories.
```

**Review by sbrl — COMMENTED** [review]
Just 1 minor comment, and then it looks ok to me :-)

**Review by sbrl — APPROVED** [review]
Thanks, @gutjuri

**Pull request merged**